src/main/java/com/redhat/devtools/intellij/tektoncd/utils/WatchHandler.java

```diff
@@ -226,6 +224,7 @@ public void eventReceived(Action action, T resource) {
                 // watches for *runs are always active so there also could be nothing to refresh, only a notification to display
                 if (!SettingsState.getInstance().displayPipelineRunResultAsNotification ||
                         !(resource instanceof io.fabric8.tekton.pipeline.v1beta1.PipelineRun) ||
+                        ((PipelineRun) resource).getStatus() == null ||
```

Comment:
Not sure to understand this change

Reply:
The getStatus can return null somehow and if we don't check it,the next getCompletionTime call fails with a nullpointerex. Not sure why I didn't have it earlier.